data/mods/ssb/statuses.ts

```diff
@@ -57,6 +57,22 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
 			}
 		},
 	},
+
+	frostyicelad: {
+		noCopy: true,
+		onStart(source) {
+			this.add(`c|${getName('Frostyicelad')}|Oh i guess its my turn now! Time to sweep!`);
```

Comment:
```suggestion
			this.add(`c|${getName('frostyicelad ❆')}|Oh i guess its my turn now! Time to sweep!`);
```

Reply:
i went by what frosty put on the forums
``Name: Frostyicelad``

Reply:
spoke with them on main